actioncable/lib/rails/generators/channel/channel_generator.rb

```diff
@@ -15,12 +15,27 @@ def create_channel_file
         if options[:assets]
           template "assets/channel.coffee", File.join('app/assets/javascripts/channels', class_path, "#{file_name}.coffee")
         end
+
+        generate_application_cable_files
       end
 
       protected
         def file_name
           @_file_name ||= super.gsub(/\_channel/i, '')
         end
+
+        # FIXME: Change these files to symlinks once RubyGems 2.5.0 is required.
```

Comment:
cc @rafaelfranca since we had this discussion in Campfire -- basically, we want to have the files that are shared between the Railties and Action Cable (also applies to the Active Record stuff below) be symlinked, to avoid duplicating the same file, since this could possibly lead to inconsistencies between the two files. Symlinked files are only supported with RubyGems 2.5.x, but [Ruby 2.2.x only ships with RubyGems 2.4.x](https://github.com/ruby/ruby/blob/ruby_2_2/lib/rubygems.rb#L12), so that is why there is a fixme here. :)